djangocms_version_locking/cms_config.py

```diff
@@ -31,5 +55,9 @@ class VersionLockingCMSConfig(CMSAppConfig):
     """
     djangocms_version_locking_enabled = getattr(
         settings, 'VERSION_LOCKING_CMS_MODELS_ENABLED', True)
-    version_lock_models = [PageContent,]
+    version_lock_models = [PageContent, ]
 
+    # Versioning config
```

Comment:
This shouldn't be necessary (as in this app you're providing core settings for locking, not versioning)

Reply:
Removing the versioning config causes an error. I didn't get round to seeing why. 

  File "./djangocms_version_locking/cms_config.py", line 48, in configure_app
    self.handle_settings(cms_config)
  File "./djangocms_version_locking/cms_config.py", line 41, in handle_settings
    raise ImproperlyConfigured('djangocms-versioning is not enabled.')
django.core.exceptions.ImproperlyConfigured: djangocms-versioning is not enabled.

Reply:
Yeah, not sure if that check is necessary/helpful. I've noticed there's a similar check if moderation, and it might be unnecessary. Moderation also checks if specified models are registered with versioning though, which you might end up using.

https://github.com/divio/djangocms-moderation/blob/release/1.0.x/djangocms_moderation/cms_config.py#L22